api/src/main/java/org/confeti/service/dto/Speaker.java

```diff
@@ -28,7 +31,7 @@
 public class Speaker implements Serializable {
 
     private static final long serialVersionUID = 1L;
-
+    @JsonIgnore
```

Comment:
` @JsonIgnore` is used to ignore the logical property used in serialization and deserialization. But we need this field in serialization. And I think we should add the test that this field is ignored during the deserialization.